src/commonMain/kotlin/baaahs/gl/render/ComponentRenderEngine.kt

```diff
@@ -126,7 +126,8 @@ class ComponentRenderEngine(
     // This must be run from within a GL context.
     private fun incorporateNewFixtures() {
         if (renderTargetsToRemove.isNotEmpty()) {
-//            TODO("remove TBD")
+//            TODO(question for xian): ("remove TBD"), how do we do this?
```

Comment:
To do it properly and condense the render buffer to remove the fixture's pixels, I think we'd want to create a new `Arrangement` sans the removed fixtures, change all the `FixtureRenderTarget`s to point at their new offsets in the render buffer, and recalculate `nextComponentOffset` and `nextRectOffset`.

Not sure it's worth it if we're usually just resetting a few brains, but I spose it'd add up quickly if we power cycle the whole bunch a few times.

Reply:
Also, it looks like the old fixtures are removed from the render list, so there's not really any render overhead for the removed fixtures, just gaps in the render buffer.

After several Reset All's:
<img width="112" alt="image" src="https://github.com/user-attachments/assets/11726c19-04c6-49a6-8a06-1587e4ee452b">